lib/offsite_payments/integrations/hi_trust.rb

```diff
@@ -31,7 +31,7 @@ def initialize(order, account, options = {})
           add_field('Type', 'Auth')
 
           # Capture the payment right away
-          add_field('depositflag', '1')
+          add_field('depositflag', '0')
```

Comment:
You currently cannot pass `depositflag` in the options as its [not whitelisted](https://github.com/Shopify/offsite_payments/blob/master/lib/offsite_payments/helper.rb#L19). Do you recommend I whitelist it?